src/main/webapp/app/modules/home/filter-activity.tsx

```diff
@@ -430,12 +458,23 @@ export class FilterActivity extends React.Component<IFilterActivityProps, IFilte
   }
 }
 
+function getTaxonomyOptions(taxonomyMap) {
+  const taxonomyOptions = {};
+  for (const key in taxonomyMap) {
```

Comment:
Can we use either .forEach or _.forEach here? Whichever is being used mostly in the project?

Reply:
we wouldn't be able to return from inside of the loop